### ISSUE TITLE
Damage dome

### ISSUE THREAD

**Issue by distributer-on-discord** [Anuken/Mindustry-Suggestions#2986]
### Describe the content or mechanics you are proposing.

A block similar to the overdrive dome, but in it's radius, friendly units/turrets do more damage, maybe increase the heal of menders and support units as well

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

The block would add more late game balance, making the larger units slightly easier to take out, letting you heal things better to protect more, and it could be balanced costing maybe a good bit of surge alloy and phase fabric.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by NemesisTheory** [comment]
Oxynoe already has overclock, which increases base damage by **115%**, base speed by **115%**, and base reload speed by **125%**. I see no reason to add this excluding either when a map has no water or anything like that.

**Comment by Ymabert** [comment]
@distributer-on-discord it makes more sense to create a new type of Robot with this support function, perhaps even with the possibility of transporting and supplying ammunition.

**Comment by OnyxFinger** [comment]
> Oxynoe already has overclock, which increases base damage by **115%**, base speed by **115%**, and base reload speed by **125%**. I see no reason to add this excluding either when a map has no water or anything like that.

that's for units tho

overall, I don't find it bad, but I think it should have difficult to satisfy needs

**Comment by Kyllingene** [comment]
>  increases base damage by 115%

Careful how you speak; it's a *multiplier* of 115%, which *increases* it by 15%.

**Comment by begginerr** [comment]
i tbh dont see a reason for this, the reason is because overdrive projector/dome exist. with them, you can make towers shoot faster and make menders heal faster. it dosent increase damage, but it increases firerate which increases dps. it dosent support units, but only oxynoe can do that as far as i know.

**Comment by OnyxFinger** [comment]
> i tbh dont see a reason for this, the reason is because overdrive projector/dome exist. with them, you can make towers shoot faster and make menders heal faster. it dosent increase damage, but it increases firerate which increases dps. it dosent support units, but only oxynoe can do that as far as i know.

but they consume more ammo

**Comment by 50275** [comment]
what's stopping you from doming your entire base

> > i tbh dont see a reason for this, the reason is because overdrive projector/dome exist. with them, you can make towers shoot faster and make menders heal faster. it dosent increase damage, but it increases firerate which increases dps. it dosent support units, but only oxynoe can do that as far as i know.
> 
> but they consume more ammo

**Comment by distributer-on-discord** [comment]
The same thing that stops you from doming your base normally, common sense?
And it would only work fir friendly units, a mender won't heal enemy
buildings

**Comment by begginerr** [comment]
> > i tbh dont see a reason for this, the reason is because overdrive projector/dome exist. with them, you can make towers shoot faster and make menders heal faster. it dosent increase damage, but it increases firerate which increases dps. it dosent support units, but only oxynoe can do that as far as i know.
> 
> but they consume more ammo

yes they do, but with those same domes you can overdrive the supply

**Comment by begginerr** [comment]
doming your entire base would... be a mess. not only would you need to supply them all, but 1 dome uses 600 power a second! not even including the factories you need! (i know that dome logic exists, but still, you need enough core supply for domes to be up) but like 2 or 3 are not bad. by the time you can make them, you most likely have good power supply to keep the domes up

**Comment by OnyxFinger** [comment]
> > > i tbh dont see a reason for this, the reason is because overdrive projector/dome exist. with them, you can make towers shoot faster and make menders heal faster. it dosent increase damage, but it increases firerate which increases dps. it dosent support units, but only oxynoe can do that as far as i know.
> > 
> > 
> > but they consume more ammo
> 
> yes they do, but with those same domes you can overdrive the supply

yup, but it costs power, and even if you OD the power you can make up for the more consumtion of the domes except that you can't because you need said domes to increase your power, which take more power

**Comment by begginerr** [comment]
> > > > i tbh dont see a reason for this, the reason is because overdrive projector/dome exist. with them, you can make towers shoot faster and make menders heal faster. it dosent increase damage, but it increases firerate which increases dps. it dosent support units, but only oxynoe can do that as far as i know.
> > > 
> > > 
> > > but they consume more ammo
> > 
> > 
> > yes they do, but with those same domes you can overdrive the supply
> 
> yup, but it costs power, and even if you OD the power you can make up for the more consumtion of the domes except that you can't because you need said domes to increase your power, which take more power

if the power you make after overdrive- the power before the overdrive is more than overdrive cost, its fine

**Comment by OnyxFinger** [comment]
> > > > > i tbh dont see a reason for this, the reason is because overdrive projector/dome exist. with them, you can make towers shoot faster and make menders heal faster. it dosent increase damage, but it increases firerate which increases dps. it dosent support units, but only oxynoe can do that as far as i know.
> > > > 
> > > > 
> > > > but they consume more ammo
> > > 
> > > 
> > > yes they do, but with those same domes you can overdrive the supply
> > 
> > 
> > yup, but it costs power, and even if you OD the power you can make up for the more consumtion of the domes except that you can't because you need said domes to increase your power, which take more power
> 
> if the power you make after overdrive minus the power before the overdrive is more than overdrive cost, its fine

unless everything takes more power, which is the case if you OD all of the prod and defense
else you may be left, if you are making a ton of power, with a bit more power, but not much. Of course, I am including all of the map's ressources being used efficiently before the OD spam.
